js/bootstrap-datepicker.js

```diff
@@ -201,6 +201,9 @@
 			}
 			o.multidateSeparator = String(o.multidateSeparator);
 
+			// We need to set the default here as the default option values are set before the DOM has loaded,
+			// meaning currentUser hasn't loaded before it's attempted to be used
+			o.weekStart = window.currentUser.startDay || 0,
```

Comment:
Is 0 Monday? Is that what it defaults to at the moment?

Reply:
Also, it feels wrong to have code that references currentUser in this library? Can't we always set `weekStart` when we call this datepicker instead?

Reply:
From the amount of files I think it is going to take some effort to decouple this.

Reply:
Ok, can we at least check if currentUser is present, just in case. `window.currentUser?.startDay`